### ISSUE TITLE
doesn't work

### ISSUE THREAD

**Issue by Elwalloon** [jqlang/jq#2867]
I just downloaded it using winget, but when i'm using it (curl http://94.237.59.206:40877/api.php/city/London | jq
) I got an errer message that says jq isn't a recognize as a command.  ('jq' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.)

Somebody knows why please ?

**Comment by nicowilliams** [comment]
You need to set the `PATH` to include the install location, or start a new `cmd` or PowerShell window.

**Comment by Elwalloon** [comment]
I still doesn't work when I restart cmd/PowerSHell window. Do I have to set a path while using Winget ? If yes, how can I do it please

**Comment by nicowilliams** [comment]
@Elwalloon see https://github.com/jqlang/jq/issues/2269#issuecomment-1235808201 and the rest of that issue.

**Comment by Elwalloon** [comment]
I finally resolved my problem, it took me hours. Thanks for your help, I got the solution while looking at #2269.

**Comment by nicowilliams** [comment]
@Elwalloon was it just setting `set PATH=%PATH%;%USERPROFILE%\appdata\local\microsoft\winget\links`, or was there more to it?

**Comment by Elwalloon** [comment]
I found my answer on https://github.com/microsoft/winget-cli/issues/2498, a link somebody said in #2269. 
I exaclty did what jamiebarrow says. I tried first without uninstalling it and it didn't work. So if somebody have the same problem, I suggest you to first  UNINSTALL jq with Winget uninstall --id jqlang.jq .

Then I did exactly that (text from jamiebarrow) :

 Steps are the same as I initially reported, but will re-list them here in case of confusion:

Launch Administrator PowerShell
winget install --id stedolan.jq (now the name is jqlang.jq)
Close Administrator PowerShell
Open new user Command Prompt
jq is not on path, close this Command Prompt
Open Advanced System Settings > Environment Variables, Press 'OK' twice to close both dialogs, making no changes to PATH as it was already updated in step (2) above
Open new user Command Prompt
jq does execute correctly, as only now PATH is effective
Also reproduced the above when running cmd as Administrator in step (1) and (3).
